### ISSUE TITLE
Add reduce motion preference

### ISSUE THREAD

**Issue by lucianmurmurache** [activist-org/activist#722]
### Terms

- [X] I have searched all [open bug reports](https://github.com/activist-org/activist/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### This is more of a missing accessibility feature but will report it as a bug
Users that have motion sickness and disable animations on their device, would expect not to see any animations on the platform. We can make use of the [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) feature which is widely supported, when applicable, we can use the [tailwind class](https://tailwindcss.com/docs/transition-property#prefers-reduced-motion) instead. 

### How to enable and test:
- Windows 10: Start > Settings > Ease of Access > Display > Toggle “Show Animations in Windows.”
- macOS: System Preferences > Accessibility > Display> Check “Reduce Motion.”
- iOS: Settings > Accessibility > Motion> Toggle “Reduce Motion.”
- Android: Settings > Accessibility > Toggle “Remove Animations.”

**Comment by andrewtavis** [comment]
Awesome, @lucianmurmurache!! Thanks so much for this :) :)

**Comment by andrewtavis** [comment]
065e7b8 adds some directions to the contributing guide for this, @lucianmurmurache 😊 Feel free to edit this section as you see fit! Thanks again for all of this consideration on your part :) :)
